### PR TITLE
Bump r-vegan to 2.5-4

### PR DESCRIPTION
**Pull request by apcamargo**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by apcamargo** [comment]
Hey @conda-forge/r-vegan ! What may be going on here? Is there any problem with the CI of this recipe?

**Comment by ocefpaf** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -61,10 +59,8 @@ about:
   summary: Ordination methods, diversity analysis and other functions for community and vegetation
     ecologists.
   license_family: GPL2
+  license_file: '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-2'
 
 extra:
   recipe-maintainers:
-    - daf
-    - ocefpaf
```
Comment (ocefpaf) [inline recipe/meta.yaml:68]: It is OK to remove me, I have universal access as a core member anyway, but I don't believe everybody you removed are in the `conda-forge/r` team.

Comment (daf) [reply recipe/meta.yaml:68]: I'm fine being removed as well but am happy to stay on if needed.  All I usually do is merge when CI looks happy.

Comment (apcamargo) [reply recipe/meta.yaml:68]: @ocefpaf I forgot to check that. In any case, it seems that only you and @daf are not part of `conda-forge/r`.

**Comment by apcamargo** [comment]
@conda-forge/r-vegan Can you guys take a look at this PR?

**Comment by ocefpaf** [comment]
@conda-forge-admin, please rerender

**Pull request merged**